src/api/java/com/enderio/api/capacitor/Scalers.java

```diff
@@ -7,6 +7,7 @@ public enum Scalers implements IScaler {
     FIXED((v, l) -> v),
     LINEAR((v, l) -> v * l),
     POW((v, l) -> (float)Math.pow(v, l)),
+    QUADRATIC((v, l) -> (float)Math.pow(v*l, 2)),
```

Comment:
Is that really the right formula?
DEV_ENERGY_TRANSFER has 120 as base value, which would be for 
level 1: 14400 FE
level 2: 57600 FE
level 3: 129600 FE
I guess the correct formula would be `v * Math.pow(l,2)`
but the normal pow has the same "problem", we might need to revisit that

Reply:
@ferriarnus Can you check if this is intended?

Reply:
Hmm could be. I added it a while ago based on 1.12, but it seems agnor is right.